zulip_bots/zulip_bots/game_handler.py

```diff
@@ -646,7 +647,7 @@ def get_game_id_by_email(self, user_email: str) -> str:
         return ''
 
     def get_bot_username(self) -> str:
-        return self.bot_handler.full_name
+        return self.full_name
```

Comment:
This change looks correct, but doesn't seem to be needed for this commit. If you want, you can make a new commit with it.